src/components/forms/ProjectGrantsForm.tsx

```diff
@@ -0,0 +1,467 @@
+            <PageText display='inline' fontSize='input'>
+              Country
+            </PageText>
+          </FormLabel>
+
+          <Select
+            id='country'
+            options={COUNTRY_OPTIONS}
+            components={{ DropdownIndicator }}
+            placeholder='Select'
+            closeMenuOnSelect={true}
+            selectedOptionColor='brand.option'
+            chakraStyles={chakraStyles}
+          />
+        </FormControl>
+
+        <FormControl isRequired mb={8}>
+          <FormLabel htmlFor='timezone'>
+            <PageText display='inline' fontSize='input'>
+              Your Time Zone
+            </PageText>
+          </FormLabel>
+
+          <Select
+            id='timezone'
+            options={TIMEZONE_OPTIONS}
+            components={{ DropdownIndicator }}
+            placeholder='Select'
+            closeMenuOnSelect={true}
+            selectedOptionColor='brand.option'
+            chakraStyles={chakraStyles}
+          />
+        </FormControl>
+
+        <FormControl isRequired mb={8}>
+          <FormLabel htmlFor='howDidYouHearAboutESP'>
+            <PageText display='inline' fontSize='input'>
+              How did you hear about the Ecosystem Support Program?
+            </PageText>
+          </FormLabel>
+
+          <Select
+            id='howDidYouHearAboutESP'
+            options={HOW_DID_YOU_HEAR_ABOUT_ESP_OPTIONS}
+            components={{ DropdownIndicator }}
+            placeholder='Select'
+            closeMenuOnSelect={true}
+            selectedOptionColor='brand.option'
+            chakraStyles={chakraStyles}
+          />
+        </FormControl>
+
+        <FormControl mb={12}>
+          <FormLabel htmlFor='recommendation'>
+            <PageText fontSize='input'>
+              Did anyone recommend that you contact Ecosystem Support?
+            </PageText>
+          </FormLabel>
+          <Input
+            id='recommendation'
+            type='text'
+            placeholder='Enter your recommendation'
+            bg='white'
+            borderRadius={0}
+            borderColor='brand.border'
+            h='56px'
+            _placeholder={{ fontSize: 'input' }}
+            color='brand.paragraph'
+            fontSize='input'
+          />
+        </FormControl>
+
+        <Center mb={12}>
+          <Flex
+            h='136px'
+            w='100%'
+            bgColor='brand.uploadProposal'
+            justifyContent='center'
+            alignItems='center'
+            cursor='pointer'
+          >
+            <Box mr={5}>
+              <Image src={uploadSVG} alt='Upload file' height={42} width={44} />
+            </Box>
+
+            <PageText fontSize='input' fontWeight={400} lineHeight='21px' maxW='220px'>
+              Upload the proposal. Click here or drag file to this box.
+            </PageText>
+          </Flex>
+        </Center>
+
+        <Center>
+          <Box position='relative'>
```

Comment:
Nice to have. Wrap all this button logic in a new button component 👍🏼

Reply:
Agree, I set a [story for that](https://www.notion.so/efdn/f4e910fa39c5411f805466aca2cadf7d?v=f90fa873a26c43318468890d4bf6d6c0&p=ed43cc43c1ac4978826cd6e332ae0b40), will apply this refactor on a future PR 👍